nautobot_chatops/workers/nautobot.py

```diff
@@ -380,20 +380,30 @@ def get_interface_connections(dispatcher, filter_type, filter_value_1, filter_va
         if ("device" or "site") in filter_type:
             # Since the device filter prompts the user to pick a site first in order to further query devices located in the chosen site, the device filter will start off with choices of all the sites with one or more devices.
             choices = [
-                (site.name, site.slug) for site in Site.objects.annotate(Count("devices")).filter(devices__count__gt=0)
+                (site.name, site.slug)
+                for site in Site.objects.annotate(Count("devices"))
+                .filter(devices__count__gt=0)
+                .order_by("name", "name")
```

Comment:
Why order by name twice? `.order_by("name")` should suffice.

Reply:
This was simply a misunderstanding on my part of the `.order_by()` call based on referencing previous instances of it.  This has been updated accordingly.